libs/design-system-components/src/components/NotificationsCard/index.tsx

```diff
@@ -39,15 +39,15 @@ const NotificationsCard: React.FC<NotificationsCardProps> = props => {
   const {
     notifications,
     isFetching,
-    followingLabel,
-    mentionedPostLabel,
-    mentionedCommentLabel,
-    replyToPostLabel,
-    replyToReplyLabel,
-    repostLabel,
-    moderatedPostLabel,
-    moderatedReplyLabel,
-    moderatedAccountLabel,
+    mentionedPostLabel = 'mentioned you in a post',
+    mentionedCommentLabel = 'mentioned you in a comment',
+    replyToPostLabel = 'replied to your post',
+    replyToReplyLabel = 'replied to your reply',
+    followingLabel = 'is now following you',
+    repostLabel = 'reposted your post',
+    moderatedPostLabel = 'moderated your post',
+    moderatedReplyLabel = 'moderated your reply',
+    moderatedAccountLabel = 'suspended your account',
     emptyTitle,
```

Comment:
having these values set here as default, will make them not get translated if missed. could be better if those label props are not optional?